inc/targetticket.class.php

```diff
@@ -1318,11 +1317,9 @@ public function save(PluginFormcreatorFormanswer $formanswer)
       ));
 
       // Add actors to ticket
-      $query = "SELECT id, actor_role, actor_type, actor_value, use_notification
-                FROM glpi_plugin_formcreator_targettickets_actors
-                WHERE plugin_formcreator_targettickets_id = " . $this->getID();
-      $result = $DB->query($query);
-      while ($actor = $DB->fetch_array($result)) {
+      $targetTicketActor = new PluginFormcreatorTargetTicket_Actor();
+      $rows = $targetTicketActor->find("`plugin_formcreator_targettickets_id` = '$targetTicketId'");
```

Comment:
yes this is wanted because I dislike strinig concatenation : it makes SQL less readable in my point of view.